contracts/test/strategies/meta.fork-test.js

```diff
@@ -0,0 +1,283 @@
+const { expect } = require("chai");
+
+const { loadFixture } = require("ethereum-waffle");
+const { isForkTest, units, ousdUnits } = require("../helpers");
+const {
+  withBalancedMetaPool,
+  withCRV3TitledMetapool,
+  withOUSDTitledMetapool,
+} = require("../_metastrategies-fixtures");
+
+// Ugly hack to avoid running these tests when running `npx hardhat test` directly.
+// A right way would be to add suffix to files and use patterns to filter
+const forkDescribe = isForkTest ? describe : describe.skip;
+
+forkDescribe("Convex 3pool/OUSD Meta Strategy", function () {
+  this.timeout(0);
+
+  describe("Balanced metapool", () => {
+    let fixture;
+    beforeEach(async () => {
+      fixture = await loadFixture(withBalancedMetaPool);
+    });
+
+    describe("Mint", function () {
+      async function mintTest(user, asset, amount = "30000") {
+        const { vault, ousd, usdt, usdc } = fixture;
+
+        const unitAmount = await units(amount, asset);
+
+        const currentSupply = await ousd.totalSupply();
+        const currentBalance = await ousd.connect(user).balanceOf(user.address);
+
+        // Mint OUSD w/ asset
+        await vault.connect(user).mint(asset.address, unitAmount, 0);
+        await vault.connect(user).allocate();
+
+        // Ensure user has correct balance (w/ 1% slippage tolerance)
+        const newBalance = await ousd.connect(user).balanceOf(user.address);
+        const balanceDiff = newBalance.sub(currentBalance);
+        expect(balanceDiff).to.approxEqualTolerance(ousdUnits(amount), 1);
+
+        // Supply checks
+        const newSupply = await ousd.totalSupply();
+        const supplyDiff = newSupply.sub(currentSupply);
+        if ([usdt.address, usdc.address].includes(asset.address)) {
+          // Ensure 2x OUSD has been added to supply
+          // (in case of USDT/USDC)
+          expect(supplyDiff).to.approxEqualTolerance(
+            ousdUnits(amount).mul(2),
+            1
+          );
+        } else {
+          // 1x for DAI
+          expect(supplyDiff).to.approxEqualTolerance(ousdUnits(amount), 1);
+        }
+
+        // // TODO: Check why this is zero on fork for USDT??
+        // const { ousdMetaPool, cvxBooster } = fixture
+        // console.log(await ousdMetaPool.connect(user).balanceOf(cvxBooster.address))
```

Comment:
I think the metastrategy should be having some balance in cvxBooster instead of metapool

Reply:
Thanks for catching this. It's actually the reward pool that should have balance of meta strategy contract, since the balance is staked when deposited. Will update the tests